pkg/internal/config/config.go

```diff
@@ -35,6 +35,9 @@ const (
 )
 
 const (
+	VMIUsername = "cloud-user"
+	VMIPassword = "0tli-pxem-xknu" // #nosec
```

Comment:
Does the first commit compiles?
AFAIU, the first commit (the one adding cloud init data) uses these consts that are introduces here.

Reply:
nope. I will move it to the first commit

Reply:
DONE